### ISSUE TITLE
404 on base route

### ISSUE THREAD

**Issue by andrewtavis** [activist-org/activist#962]
### Terms

- [X] I have searched all [open bug reports](https://github.com/activist-org/activist/issues?q=is%3Aopen+is%3Aissue+label%3Abug)
- [X] I agree to follow activist's [Code of Conduct](https://github.com/activist-org/activist/blob/main/.github/CODE_OF_CONDUCT.md)

### Behavior

Something that was realized recently is that we're for some reason getting a 404 on the base route of activist. This was introduced recently, but we're not sure when. Basically if you go to activist.org, you get a 404, and all you need to do is reload the page to get to the appropriate route given your locale. So reloading will for example take you to activist.org/en/.

CC @arescrimson

**Comment by andrewtavis** [comment]
@arescrimson: Problem was introduced in https://github.com/activist-org/activist/pull/957

**Comment by andrewtavis** [comment]
Or specifically https://github.com/activist-org/activist/pull/942, as this was merged and then the other one has the problem. It wasn't introduced by making the versions explicit, but by the upgrade from Nuxt to a later version.

**Comment by andrewtavis** [comment]
Another problem could be that the localization route just isn't being done correctly?

**Comment by andrewtavis** [comment]
@arescrimson, this is fixed locally right now, and I'm going through final tests atm. Seemed to have been something with Nuxt i18n, where updating it actually fixed the bug directly. I'm also updating Nuxt to the latest version as well :)

**Comment by andrewtavis** [comment]
Ok, well, on the plus side the route is fixed during deployment based on the changes above, but on the negative side it's broken in deployment still...

**Comment by andrewtavis** [comment]
Based on this, it's a Netlify error

**Comment by andrewtavis** [comment]
Ok this is fixed for now. Reverted one of the earliest decisions for activist, which is that all language's routes should be localized such that there's no "default" in the UI. This is breaking on Netlify's end at the moment, so I've set the i18n strategy to `prefix_and_default` meaning that `en` routes work if they're prefixed and not.

Thanks for the suggestion to get this up, @arescrimson! Sorry to jump in, but this was definitely unbreak now level...

Let me know what of the issues we discussed you'd have interest in!
